src/main/java/com/example/demo/mappers/UserEntityMapper.java

```diff
@@ -4,17 +4,20 @@
 import com.example.demo.model.dto.UserCreation;
 import com.example.demo.model.dto.UserUpdate;
 import com.example.demo.model.entities.UserEntity;
+import org.hibernate.annotations.CreationTimestamp;
 import org.springframework.stereotype.Component;
 
+import java.time.Instant;
 import java.time.LocalDateTime;
 import java.util.Date;
 
 @Component
 public class UserEntityMapper {
+
     public UserEntity map(UserCreation userCreation) {
-        return new UserEntity(userCreation.getName(), userCreation.getAge(), new Date(), new Date(), Role.USER, userCreation.getPassword());
+        return new UserEntity(userCreation.getName(), userCreation.getAge(), LocalDateTime.now(), LocalDateTime.now(), Role.USER, userCreation.getPassword());
```

Comment:
you need to add  LocalDateTime.now() as you use @creationtimestamp and @updateTimestamp . Hibernate will do work for you